shortstories/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+class ShortStory(models.Model):
+     status=(
+        ('dr', 'Draft'),
+        ('p' , 'Published'),
+        ('b' , 'banned'),
```

Comment:
plz be consistent in your coding style and Capitalize your first word

Reply:
done